### DISCUSSION TITLE
how to save view by Exploded View sample

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#3815] [Q&A] [answered]
use the Exploded View sample
how to save the  view ( for example the Explode Scaling = 2 ) and load next time  ??

![exploderview](https://user-images.githubusercontent.com/18022720/173816047-40d89bf4-00fe-4165-b829-66cd60aaafda.png)

**Comment by pmconne** [comment, marked as answer]
The sample does not support that. You would need to write your own code that saves the current state and recalls it later.

**Reply by paprophet** [reply]
> The sample does not support that. You would need to write your own code that saves the current state and recalls it later.
//saveview
 const vp = IModelApp.viewManager.selectedView!.view;
        const viewProp = ViewStateHelper.viewStateToProps(vp);
       ViewStateHelper.emphasizeElementsToProps(IModelApp.viewManager.selectedView!, viewProp)

//load
  const viewState = await ViewStateHelper.viewStateFromProps(imodel, viewProp);
                if (viewState) {
                    vp.changeView(viewState);
                }

but in Exploded View sample, i use this code ,but the view have no explosion effect  ??

**Reply by pmconne** [reply]
Correct, you should not expect the explosion effect to be saved as part of the view. It is a sample. If you want to save the effect, you must **write your own code** to do that.

**Reply by ayushibasaiwal** [reply]
@paprophet If your query is resolved, can you please mark this reply as an answer? or in case you need any help, please ask.
